proto/plugin.proto

```diff
@@ -804,3 +815,34 @@ message ConfigSource {
     }
   }
 }
+
+/********************************************************************
+* TaskLauncher Plugin Service
+********************************************************************/
+
+// TaskLauncher service to start and stop task (activities spawned from
+// oci images that run until completion and are not restarted) that take
+// some set of arguments and returns it's identification state.
+service TaskLauncher {
+  rpc IsAuthenticator(google.protobuf.Empty) returns (ImplementsResp);
+  rpc Auth(FuncSpec.Args) returns (Auth.AuthResponse);
+  rpc AuthSpec(google.protobuf.Empty) returns (FuncSpec);
+  rpc ValidateAuth(FuncSpec.Args) returns (google.protobuf.Empty);
+  rpc ValidateAuthSpec(google.protobuf.Empty) returns (FuncSpec);
```

Comment:
We can probably dump the auth stuff for task launchers for now, especially since auth is like... uncertain at the moment? Or no?

Reply:
Ah good call. Just came over in as copypasta.